### PR TITLE
refactor: upgrade node standards

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #94** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/94" target="_blank">Graphite</a>  👈
    * **PR #95** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/95" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/94?utm_source=stack-comment).

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/94?utm_source=stack-comment).


Job Reference: h3oynYMWyzSUkgjE0Mhc

**Pull request merged**
